TestPlaybooks/playbook-listExecutedCommands-Test.yml

```diff
@@ -0,0 +1,243 @@
+      }
+  "3":
+    id: "3"
+    taskid: bc5eb7ee-ace6-4541-8f41-ecff7c043efe
+    type: regular
+    task:
+      id: bc5eb7ee-ace6-4541-8f41-ecff7c043efe
+      version: -1
+      name: Save response to context
+      description: "Takes the comments of a given entry ID and stores them in the
+        incident context, under a provided context key. \nFor accessing the last executed
+        task's comments, provide ${lastCompletedTaskEntries.[0]} as the value for
+        the entryId input parameter."
+      scriptName: commentsToContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      contextKey: noData
+      entryId: ${lastCompletedTaskEntries}
+      listSeperator: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 545
+        }
+      }
+  "4":
+    id: "4"
+    taskid: 8629b4dc-1921-401a-8315-2ac29b4504d1
+    type: condition
+    task:
+      id: 8629b4dc-1921-401a-8315-2ac29b4504d1
+      version: -1
+      name: Check that we didn't get any commands
+      scriptName: AreValuesEqual
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "5"
+    scriptarguments:
+      left: ${noData=val.trim()}
+      right: No commands found
+    results:
+    - AreValuesEqual
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 719
+        }
+      }
+  "5":
+    id: "5"
+    taskid: 28c54813-8890-461e-8839-d70962f93a2e
+    type: regular
+    task:
+      id: 28c54813-8890-461e-8839-d70962f93a2e
+      version: -1
+      name: Get playbook commands
+      scriptName: listExecutedCommands
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      source: Playbook
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 895
+        }
+      }
+  "6":
+    id: "6"
+    taskid: 5fd8f086-ce66-47c0-8677-5802caf863b6
+    type: regular
+    task:
+      id: 5fd8f086-ce66-47c0-8677-5802caf863b6
+      version: -1
+      name: Save response to context
+      scriptName: commentsToContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      contextKey: withData
+      entryId: ${lastCompletedTaskEntries}
+      listSeperator: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1069
+        }
+      }
+  "7":
+    id: "7"
+    taskid: 4d15e900-cce0-4f9e-815b-4a01bea70368
+    type: condition
+    task:
+      id: 4d15e900-cce0-4f9e-815b-4a01bea70368
+      version: -1
+      name: Check that we got commands
+      scriptName: AreValuesEqual
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "8"
+    scriptarguments:
+      left: ${withData=val.trim()}
+      right: No commands found
```

Comment:
Shouldn't it find results in this case (The playbook `Print` task)?

Reply:
So first time I'm calling it in the playbook, i use the source=manual, so it shouldn't find. The second time I run it, I use source=playbook. and the playbook checks those conditions.

Reply:
and you navigate only on the "no" condition. saw it just now.